fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocksLex.g4

```diff
@@ -25,6 +25,7 @@ tokens {
 }
 
 ACCESS: 'ACCESS';
+ACTIVE: 'ACTIVE';
```

Comment:
shall we use `activate` instead of `active`, activate is a verb, but active is a adj j

Reply:
I think active is better.
active is relatively concise, and activate is a bit like materialized, which is easy to misspell.
Or we can also consider other words, such as ENABLE, RESUME?